cluster/sync.sh

```diff
@@ -52,3 +52,4 @@ pods_ready_wait() {
 
 pods_ready_wait
 make create-nodeport
+./cluster/kubectl.sh apply -f ./hack/psa/kubevirt.yaml
```

Comment:
please create a function for it, like pods_ready_wait

Reply:
done.